charmhelpers/contrib/openstack/context.py

```diff
@@ -489,7 +489,7 @@ def __call__(self):
             ha_vip_only = False
             self.related = True
             transport_hosts = None
-            rabbitmq_port = '5672'
+            rabbitmq_port = 5672
```

Comment:
Why did this change from a string to a number?  I'm worried that, as this is a library function, the change in type may have consequences for other code that might be depending on it being a string?